benchmarks/perception/a3_stereo_image_proc/config/empty.yaml

```diff
@@ -0,0 +1 @@
+# This is an empty file, so that git commits the folder correctly
```

Comment:
Why do you need this folder under version control if there is no meaningful content in it?

Reply:
Yeah, that was some old code, I'll delete it, thanks!